src/binary-reader-objdump.cc

```diff
@@ -57,12 +60,17 @@ class BinaryReaderObjdumpBase : public BinaryReaderNop {
   bool print_details = false;
   BinarySection reloc_section = BinarySection::Invalid;
   uint32_t section_starts[kBinarySectionCount];
+  bool section_found = false;
+  Stream* out_stream = nullptr;
 };
 
 BinaryReaderObjdumpBase::BinaryReaderObjdumpBase(const uint8_t* data,
                                                  size_t size,
                                                  ObjdumpOptions* options)
-    : options(options), data(data), size(size) {
+    : options(options),
+      data(data),
+      size(size),
+      out_stream(init_stdout_stream()) {
```

Comment:
Probably not a major issue, but I'm a bit worried about calling init_stdout_stream() twice. I think it would be preferrable to pass in a `Stream *` to the BinaryReaderObjdumpDisassemble constructor.